src/Simulations/VarianceDissipationComputations/update_fluxes.jl

```diff
@@ -2,11 +2,11 @@ using Oceananigans: fields
 using Oceananigans.Grids: topology, Flat
 
 # Store advective and diffusive fluxes for dissipation computation
-function cache_fluxes!(model, dissipation)
-    grid   = model.grid
-    arch   = architecture(grid)
-    sz     = size(model.tracers[1].data)
-    of     = model.tracers[1].data.offsets
+function cache_fluxes!(model, dissipation, tracer_name::Symbol)
```

Comment:
why dispatch? what happens in the other case?

Reply:
there should be no other case, I think that remained there from earlier dispatch attempts, I can remove it